docs/components/store.md

```diff
@@ -296,9 +296,15 @@ Currently only memcached "backend" is supported:
 ```yaml
 type: MEMCACHED # Case-insensitive
 config:
-  addresses:
-    - localhost:11211
-
+  addresses: []
```

Comment:
Hm... Ideally we could autogenerate this 🤗  but we can update to Giedrius proposed thing first and automate at some point